pkg/api/aim/runs.go

```diff
@@ -412,8 +412,8 @@ func SearchRuns(c *fiber.Ctx) error {
 			ID: q.Offset,
 		}
 		// TODO:DSuhinin -> do we need `namespace` restriction? it seems like yyyyess, but ....
-		if tx = database.DB.Select("row_num").First(&run); tx.Error != nil && tx.Error != gorm.ErrRecordNotFound {
-			return fmt.Errorf("unable to find search runs offset %q: %w", q.Offset, tx.Error)
+		if err = database.DB.Select("row_num").First(&run).Error; err != nil && tx.Error != gorm.ErrRecordNotFound {
```

Comment:
you're using tx.Error in the if

Reply:
also let's make sure we assign to a new `err` to be on the safe side (`if err := ...`)

Reply:
Im not finished it yet.

Reply:
ok, now it is ready for review.